bigtable/google/cloud/bigtable/batcher.py

```diff
@@ -143,8 +143,15 @@ def mutate_rows(self, rows):
             self.mutate(row)
 
     def flush(self):
-        """ Sends the current. batch to Cloud Bigtable. """
-        if len(self.rows) is not 0:
+        """ Sends the current. batch to Cloud Bigtable.
```

Comment:
Please remove the period after "current".  Also, include a blank line after this one, per [PEP 257](https://www.python.org/dev/peps/pep-0257/#id17).

Reply:
This was totally inherited from some other commit, along with the errors, but agreed and done.